examples/python/xds/README.md

```diff
@@ -28,9 +28,40 @@ python server.py
 
 3. Verify the Server
 
-This step is not strictly necessary, but you can use it as a sanity check if
-you'd like. If you don't have it, install
-[`grpcurl`](https://github.com/fullstorydev/grpcurl/releases). This will allow
+After configuring your xDS server to track the gRPC server we just started,
+create a bootstrap file as desribed in [gRFC A27](https://github.com/grpc/proposal/blob/master/A27-xds-global-load-balancing.md):
+
+```
+{
+  xds_servers": [
+    {
+      "server_uri": <string containing URI of xds server>,
+      "channel_creds": [
+        {
+          "type": <string containing channel cred type>,
+          "config": <JSON object containing config for the type>
+        }
+      ]
+    }
+  ],
+  "node": <JSON form of Node proto>
+}
+```
```

Comment:
optional: Shall we create a demo bootstrap file pointing to localhost? So users who run this example can experiment with it.

Reply:
Having an xDS server on `localhost` is a very unusual case. It would only make sense in the context of testing an xDS server as a developer on that server. IMO, it would make more sense to include Traffic Director, but I was trying to keep things neutral in the README.